README.md

```diff
@@ -1,4 +1,208 @@
 eos-knowledge-lib
 =================
 
-GUI library for knowledge apps
+Welcome to **eos-knowledge-lib**!
+This is Endless' Javascript library for building offline content apps using modular UI blocks and declarative UI files.
+
+This project uses the GNOME technology stack, including GTK and WebKit for UI, and GJS for its Javascript interpreter.
+
+Building
+--------
+We recommend using [JHbuild](https://developer.gnome.org/jhbuild/stable/) to build this code, especially if you are planning to make changes to it.
+A sample JHbuild moduleset is included in the root directory of this repository.
+
+First download the moduleset with:
+```sh
+wget https://raw.githubusercontent.com/endlessm/eos-knowledge-lib/master/eos-knowledge-lib.modules
+```
+
+Then edit `~/.jhbuildrc` to get JHbuild to use the provided moduleset.
```

Comment:
The location for the jhbuildrc is in `XDG_CONFIG_HOME`, these days, so: `$HOME/.config/jhbuildrc`.